pkg/manifests/manifests.go

```diff
@@ -2602,7 +2602,7 @@ func (f *Factory) ThanosQuerierDeployment(grpcTLS *v1.Secret, enableUserWorkload
 
 	if f.config.ThanosQuerierConfig.Resources != nil {
 		for i, c := range d.Spec.Template.Spec.Containers {
-			if c.Name == "thanos-querier" {
+			if c.Name == "thanos-query" {
```

Comment:
We have this in the unit tests as well. Can you also add we should error out if that container name is not there, thanks! Or we can do that in follow up as well. Lets just adjust that test, thanks!

https://github.com/openshift/cluster-monitoring-operator/blob/98bc99134498721360fdde87ca54f9233ae4bffa/pkg/manifests/manifests_test.go#L1393

Reply:
good catch, fixed